cmd/kubeadm/app/constants/constants_windows.go

```diff
@@ -21,4 +21,10 @@ package constants
 const (
 	// DefaultDockerCRISocket defines the default Docker CRI socket
 	DefaultDockerCRISocket = "npipe:////./pipe/docker_engine"
+
+	//ImageRepository defines the repository to be used for the pause image
+	ImageRepository = "mcr.microsoft.com/k8s/core"
+
+	// PauseVersion indicates the default pause image version for kubeadm
+	PauseVersion = "1.2.0"
```

Comment:
i was under the impression that the long term goal is to make the pause image at GCR a multi-OS image and make it versioned as e.g. `3.2` to simplify the UX.
i'm hesitant to apply this change until this topic is clear.

this issue did not a give an answer to that:
https://github.com/kubernetes/kubernetes/issues/74218

also, the kubelet has previously rejected defaulting differently between OSes (for flags at least).